docs/howto/operator/google/transfer/azure_fileshare_to_gcs.rst

```diff
@@ -0,0 +1,54 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+Transfers data from Azure FileShare Storage to Google Cloud Storage
+============================================================
```

Comment:
Here will be error in doc build.
The "line" under header should have the same length as the string `Transfers data from Azure FileShare Storage to Google Cloud Storage`

Reply:
indeed